src/common/ServerCompatBanner.js

```diff
@@ -9,16 +9,24 @@ import { getIdentity, getServerVersion } from '../account/accountsSelectors';
 import { getIsAdmin, getSession, getGlobalSettings } from '../directSelectors';
 import { dismissCompatNotice } from '../session/sessionActions';
 import { openLinkWithUserPreference } from '../utils/openLink';
+import { ZulipVersion } from '../utils/zulipVersion';
 
 // The oldest version we currently support. Should match what we say at
 //   https://zulip.readthedocs.io/en/stable/overview/release-lifecycle.html#compatibility-and-upgrading.
-const minSupportedVersion = '2.1.0';
+export const kMinSupportedVersion: ZulipVersion = new ZulipVersion('2.1.0');
 // Notes on known breakage at older versions:
 //  * Before 1.8, the server doesn't send found_newest / found_oldest on
 //    fetching messages, and so `state.caughtUp` will never have truthy
 //    values.  This probably means annoying behavior in a message list,
 //    as we keep trying to fetch newer messages.
 
+/**
+ * The next value we'll give to kMinSupportedVersion in the future.
+ *
+ * This should be the next major Zulip Server version after kMinSupportedVersion.
+ */
+export const kNextMinSupportedVersion: ZulipVersion = new ZulipVersion('3.0');
```

Comment:
Is the "k" for "constant"? 🤔

Reply:
Yeah. This is influenced by Google style:
https://google.github.io/styleguide/cppguide.html#Constant_Names

I don't have any desire to go sweeping through existing code, but I'm dissatisfied with SCREAMING_SNAKE_CASE because it's more of a pain to type, and also more different from the other names we use. So wanted to experiment with this style.